etcdctl/ctlv2/ctl.go

```diff
@@ -54,6 +56,12 @@ func Start(apiv string) {
 		cli.StringFlag{Name: "cert-file", Value: "", Usage: "identify HTTPS client using this SSL certificate file"},
 		cli.StringFlag{Name: "key-file", Value: "", Usage: "identify HTTPS client using this SSL key file"},
 		cli.StringFlag{Name: "ca-file", Value: "", Usage: "verify certificates of HTTPS-enabled servers using this CA bundle"},
+		cli.BoolFlag{Name: "insecure-skip-tls-verify", Usage: "skip server certificate verification"},
+		cli.GenericFlag{Name: "cipher-suites",
+			Value: flags.NewStringSliceFlag(tlsutil.AvailableCipherSuites()...),
+			Usage: "Comma-separated list of cipher suites for the server. " +
+				"Values are from tls package constants (https://golang.org/pkg/crypto/tls/#pkg-constants). " +
```

Comment:
suggest enumerating the values in AvailableCipherSuites() rather than referencing godoc